nodeJS/express/using_postgresql.md

```diff
@@ -0,0 +1,309 @@
+<!-- markdownlint-disable TOP004 -->
+
+### Introduction
+
+Now that we have PostgreSQl up and running on our machine, it's time to work with it.
+
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+#### Prerequisite SQL knowledge
+
+Make sure you've completed the [SQL course](https://www.theodinproject.com/paths/full-stack-javascript/courses/databases).
+
+This lesson and all subsequent lessons will assume you understand SQL syntax and concepts.
+
+</div>
+
+For brevity's sake, we'll refer to database as db from now on.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Setting up a new db in PostgreSQL shell
+- Setting up and querying with node-postgres
+- Using a script to populate the db
+
+### Setting up a Express app
+
+Let's start by creating a Express application. It will just have one feature - add usernames provided by the user to the db. Here are the expected routes and their functionalities:
+
+- `GET /` - to log available usernames in the DB to the terminal. For now, just put in a simple `console.log('usernames will be logged here - wip')`.
+- `GET /new` - to display a HTML form to the user with one username input text field. It will submit to the following route,
```

Comment:
Might look a tiny bit cleaner than a trailing comma?

```suggestion
- `GET /new` - to display a HTML form to the user with one username input text field. It will submit to the next route.
```